### ISSUE TITLE
Do not add semantics to the geometry when LoD1

### ISSUE THREAD

**Issue by balazsdukai** [cityjson/cjio#44]
When a Geometry is created and populated in LoD1, the geometries of the CityObjects in the CityJSON file contain the 'semantics' key and the values set to 'null'.

**Comment by hugoledoux** [comment]
why not? Is it because CityGML specs state so? I would argue that this should be ignored, if I get a file with LoD1 and I want to do something, it's surely useful to know which surfaces are the roofs, no?

I know it's "easy" to extract from the geometry, but it's not that easy.

**Comment by balazsdukai** [comment]
Yes that's true. Also, apparently my issue was vague, because I actually meant a different problem (see https://github.com/cityjson/cjio/issues/45). Nevertheless this helped to point out the case that you are mentioning, so I'm leaving this open.

It would still be the job of the user to label the surfaces in an LoD1 model though, right? Or are you thinking about a `label_my_surface` magic function?

**Comment by hugoledoux** [comment]
I wasn’t thinking of automatic function, but that is an idea! Rather simple to implement, let’s do that (at some point)
